### PR TITLE
Update RPTRST for 2024-04

### PR DESCRIPTION
**Pull request by gdfldm**
Update RPTRST tables to include additional arrays including:

FREQ, RESIDUAL, CONV, 

XMFCO2, XMFH2, YMFWAT

Array required to restart Polymer, MICP, Hysteresis, PRECSALT, etc. runs

### CODE REVIEW

File: parts/chapters/subsections/10.3/RPTRST.fodt [hunk 1]
```diff
@@ -4478,184 +4468,434 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
...
        <text:list xml:id="list105919273011646" text:continue-list="list105918417837053" text:style-name="_40_NumberedListTable">
         <text:list-item text:start-value="1">
          <text:p text:style-name="P19573">OPTION = 0 then output to the restart file is stopped. </text:p>
         </text:list-item>
         <text:list-item>
-         <text:p text:style-name="P19573">OPTION = 1 then the restart files are written at every report time, but only the last one in the run is kept. This minimizes the restart file size but only the final results are stored, limiting the visualization in OPM ResInsight.</text:p>
+         <text:p text:style-name="P19573">OPTION = 1 then the restart files are written at every report time, but only the last one output is kept. This minimizes the restart file size but limits the visualization in OPM ResInsight to the final conditions.</text:p>
```
Comment (blattms) [inline parts/chapters/subsections/10.3/RPTRST.fodt:4491]: "last one ouput is" reads strange: Either "last output is" or "last one is"

**Review by blattms — COMMENTED** [review]
Small nitpick:

File: parts/chapters/subsections/10.3/RPTRST.fodt (hunk 1 above)
Comment (gdfldm) [reply parts/chapters/subsections/10.3/RPTRST.fodt]: Agreed, I have gone with "last output is"

**Comment by blattms** [comment]
Perfect, thanks.

**Pull request merged**